test/nimble_template/addons/credo_test.exs

```diff
@@ -41,7 +41,11 @@ defmodule NimbleTemplate.Addons.CredoTest do
           assert file =~ """
                    defp aliases do
                      [
-                       codebase: [\"deps.unlock --check-unused\", \"format --check-formatted\", \"credo --strict\"],
+                       codebase: [
+                         "credo --strict",
+                         "deps.unlock --check-unused",
+                         "format --check-formatted"
```

Comment:
Do we need to remove this on this test based on our new update? 💡

Reply:
Yeah, we can remove these assert, I have removed it in https://github.com/nimblehq/elixir-templates/pull/144/commits/88c7a3ec017f96f8340b525456ab2a2fc55a85f8